### PR TITLE
183 county picker

### PR DESCRIPTION
**Pull request by JohnAGill**
Updated Kings, Richmond & New York & added 3 new values for more user friendly values

### CODE REVIEW

**Review by floridemai — CHANGES_REQUESTED** [review]
@JohnAGill Are you sure this work? I think the component search for the code and no matter what you select will probably display the first value foud.
I think the solution is to use `code_variant` as the code and when you need to set the value (to get the data for example) you simply remove the variant. This is safe enough to do as we know all the codes and no need to worry about `_`

File: src/providers/settings.tsx [hunk 1]
```diff
@@ -277,7 +277,7 @@ function getAgeRangeOptions(t: TFunction): AgeOption[] {
 function getCountiesOptions(t: TFunction): BasicItem[] {
   return [
     {label: t('county:notinny'), value: 'u'},
-    ...counties.map((c) => ({label: c.county, value: c.code}))
+    ...counties.map((c) => ({label: c.label ? c.label : c.county, value: c.code}))
```
Comment (AlanSl) [inline src/providers/settings.tsx]:
```suggestion
    ...counties.map((c) => ({label: c.label || c.county, value: c.code}))
```

File: src/components/molecules/county-dropdown.tsx [hunk 2]
```diff
@@ -14,8 +14,8 @@ const countyOptions: CountyOption[] = [];
 
 for (const l of counties) {
   countyOptions.push({
-    label: l.county,
-    value: l.county
+    label: l.label ? l.label : l.county,
```
Comment (AlanSl) [inline src/components/molecules/county-dropdown.tsx]:
```suggestion
    label: l.label || l.county,
```

**Review by AlanSl — COMMENTED** [review]
Looks good to me if @floridemai 's happy with the latest changes. 

I've just made a couple of non-essential suggestions where the code could be a bit cleaner

File: src/components/molecules/county-dropdown.tsx (hunk 2 above)
Comment (JohnAGill) [reply src/components/molecules/county-dropdown.tsx]: Made these changes, need @floridemai to review again for some further changes

Comment (AlanSl) [reply src/components/molecules/county-dropdown.tsx]: Thanks @JohnAGill - if you're waiting for re-review don't forget to press the re-review button next to Paul's name: 
image

**Review by floridemai — APPROVED** [review]
No need to introduce the label. You can just name the county `Brooklyn (Kings)`

**Pull request merged**